experiments/download_ssl4eo.py

```diff
@@ -519,52 +427,28 @@ def worker(idx: int) -> None:
         if str(idx) in ext_coords.keys():
             return
 
-        (patches_s1, patches_s2c, patches_s2a, center_coord) = get_random_patches_match(
+        patches, center_coord = get_random_patches_match(
             idx,
-            collections,
+            collection,
             bands,
             crops,
+            dtype,
             dates,
-            radius=radius,
+            radius=args.radius,
             debug=args.debug,
             match_coords=match_coords,
         )
 
-        if (
-            patches_s1 is not None
-            and patches_s2c is not None
-            and patches_s2a is not None
-        ):
+        if patches is not None:
             if args.save_path is not None:
-                # s2c
-                location_path_s2c = os.path.join(args.save_path, "s2c", f"{idx:06d}")
-                os.makedirs(location_path_s2c, exist_ok=True)
-                for patch in patches_s2c:
-                    save_patch(
-                        raster=patch["raster"],
-                        coords=patch["coords"],
-                        metadata=patch["metadata"],
-                        path=location_path_s2c,
-                    )
-                # s2a
-                location_path_s2a = os.path.join(args.save_path, "s2a", f"{idx:06d}")
-                os.makedirs(location_path_s2a, exist_ok=True)
-                for patch in patches_s2a:
-                    save_patch(
-                        raster=patch["raster"],
-                        coords=patch["coords"],
-                        metadata=patch["metadata"],
-                        path=location_path_s2a,
-                    )
-                # s1
-                location_path_s1 = os.path.join(args.save_path, "s1", f"{idx:06d}")
-                os.makedirs(location_path_s1, exist_ok=True)
-                for patch in patches_s1:
+                location_path = os.path.join(args.save_path, "imgs", f"{idx:06d}")
```

Comment:
Doesn't your SSL4EO-S12 dataset use 7 digits instead of 6? I would like to remain consistent if we can.

Reply:
yep, sure